app/src/minApi26/java/mod/jbk/build/BuiltInLibraries.java

```diff
@@ -0,0 +1,614 @@
+    public static String ANDROIDX_SLIDINGPANELAYOUT = "slidingpanelayout-1.1.0";
+    public static String ANDROIDX_SQLITE = "sqlite-2.2.0";
+    public static String ANDROIDX_SQLITE_FRAMEWORK = "sqlite-framework-2.2.0";
+    public static String ANDROIDX_STARTUP_RUNTIME = "startup-runtime-1.1.1";
+    public static String ANDROIDX_SWIPEREFRESHLAYOUT = "swiperefreshlayout-1.2.0-alpha01";
+    public static String ANDROIDX_TRACING = "tracing-1.0.0";
+    public static String ANDROIDX_TRANSITION = "transition-1.5.0";
+    public static String ANDROIDX_VECTORDRAWABLE = "vectordrawable-1.1.0";
+    public static String ANDROIDX_VECTORDRAWABLE_ANIMATED = "vectordrawable-animated-1.1.0";
+    public static String ANDROIDX_VERSIONEDPARCELABLE = "versionedparcelable-1.1.1";
+    public static String ANDROIDX_VIEWPAGER2 = "viewpager2-1.0.0";
+    public static String ANDROIDX_VIEWPAGER = "viewpager-1.0.0";
+    public static String ANDROIDX_WORK_RUNTIME = "work-runtime-2.7.1";
+    public static String FIREBASE_AUTH = "firebase-auth-19.0.0";
+    public static String FIREBASE_AUTH_INTEROP = "firebase-auth-interop-18.0.0";
+    public static String FIREBASE_COMMON = "firebase-common-19.3.1";
+    public static String FIREBASE_COMPONENTS = "firebase-components-16.0.0";
+    public static String FIREBASE_DATABASE = "firebase-database-19.3.1";
+    public static String FIREBASE_DATABASE_COLLECTION = "firebase-database-collection-17.0.0";
+    public static String FIREBASE_DYNAMIC_LINKS = "firebase-dynamic-links-19.0.0";
+    public static String FIREBASE_IID = "firebase-iid-19.0.0";
+    public static String FIREBASE_IID_INTEROP = "firebase-iid-interop-17.0.0";
+    public static String FIREBASE_MEASUREMENT_CONNECTOR = "firebase-measurement-connector-18.0.0";
+    public static String FIREBASE_MESSAGING = "firebase-messaging-19.0.0";
+    public static String FIREBASE_STORAGE = "firebase-storage-19.0.0";
+    public static String CIRCLE_IMAGEVIEW = "circleimageview-3.1.0";
+    public static String CODE_VIEW = "CodeView-0.4.0";
+    public static String FACEBOOK_ADS_AUDIENCE_NETWORK_SDK = "audience-network-sdk-5.9.0";
+    public static String GLIDE_ANNOTATIONS = "annotations-4.11.0";
+    public static String GLIDE_DISKLRUCACHE = "disklrucache-4.11.0";
+    public static String GLIDE_GIFDECODER = "gifdecoder-4.11.0";
+    public static String GLIDE = "glide-4.11.0";
+    public static String GOOGLE_AUTO_VALUE_ANNOTATIONS = "auto-value-annotations-1.6.5";
+    public static String GSON = "gson-2.8.7";
+    public static String GUAVA_LISTENABLEFUTURE = "listenablefuture-1.0.0";
+    public static String HTTP_LEGACY_ANDROID_28 = "http-legacy-android-28";
+    public static String JETBRAINS_ANNOTATIONS = "annotations-13.0";
+    public static String KOTLIN_STDLIB = "kotlin-stdlib-1.6.21";
+    public static String KOTLIN_STDLIB_JDK7 = "kotlin-stdlib-jdk7-1.6.21";
+    public static String KOTLIN_STDLIB_JDK8 = "kotlin-stdlib-jdk8-1.6.21";
```

Comment:
and this is just unnecessary because it's don't have any class in

Reply:
i see many internal libs have the jdk8 as a dependency so i add it do u recommend removing it ??

Reply:
> and this is just unnecessary because it's don't have any class in

as i can see it does have u can check dex, and since its an addon so the main kotlin core use it and as i told u before many libs have that as a dependencie.

Reply:
reverted ✅️